src/transformers/models/idefics/image_processing_idefics.py

```diff
@@ -92,8 +92,9 @@ def preprocess(
         image_mean: Optional[Union[float, List[float]]] = None,
         image_std: Optional[Union[float, List[float]]] = None,
         transform: Callable = None,
+        return_tensors: Optional[Union[str, TensorType]] = None,
```

Comment:
```suggestion
        return_tensors: Optional[Union[str, TensorType]] = TensorType.PYTORCH,
```

We should keep PyTorch as the default here so as not to break backward compatibility.

Reply:
Done!